app/registries/foreman/settings/general.rb

```diff
@@ -51,11 +51,11 @@
       description: N_("Whether or not to show a menu to access experimental lab features (requires reload of page)"),
       default: false,
       full_name: N_('Show Experimental Labs'))
-    setting('append_domain_name_for_hosts',
+    setting('display_fqdn_for_hosts',
       type: :boolean,
-      description: N_('Foreman will append domain names when new hosts are provisioned'),
+      description: N_('Foreman will display domain names appended to hostnames.'),
```

Comment:
```suggestion
      description: N_('Foreman will display names of hosts as FQDNs. If disabled, Foreman will only display names of hosts as hostnames.'),
```
Not sure about the terminology here. If "shortname" is a thing, then maybe say "shortnames" instead of hostnames. But to my understanding it's hostnames and I'd prefer the common industry term.

Reply:
You are spot on. I think someone came up with when naming the variable and it caught on.